### ISSUE TITLE
Error: Failed to start application: Unable to load session storage class: php

### ISSUE THREAD

**Issue by Concepto-Com** [joomla/joomla-cms#43687]
### Steps to reproduce the issue
1. enter website

### Expected result
show website

### Actual result
error

### Device and system info

* What version of Joomla were you using?
3.4

* What device were you using? (eg, phone, laptop)
laptop

* What web browser were you using? (eg, Chrome, Safari)
chrome, safari and opera

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by Quy** [comment]
Please ask support questions at https://forum.joomla.org/.
